netbox/dcim/forms.py

```diff
@@ -70,6 +70,28 @@ class Meta:
         fields = ['parent', 'name', 'slug']
 
 
+class RegionCSVForm(forms.ModelForm):
+    parent = forms.ModelChoiceField(
```

Comment:
Not sure -- in my setup where I have a region "Europe", another one "US", with children "US East" and "US West", with this change it shows as:
```
Name of parent region
Choices: Europe, US, --- US East, etc. 
```
…while before this change it shows up as:
```
 Name of parent region
Choices: Europe, US, US East, etc. 
```